xbmc/video/VideoDatabase.cpp

```diff
@@ -4680,11 +4775,22 @@ void CVideoDatabase::UpdateTables(int iVersion)
       m_pDS->next();
     }
   }
+
+  if (iVersion < 101)
+  {
+    std::string strSQL = PrepareSQL("ALTER TABLE movie ADD c%02d integer", VIDEODB_ID_MOVIE_BOOKMARK);
```

Comment:
Yes, sorry forgot about that. 
Just keep it cXX for now. It's out of scope and we can just do that in a pr later.